<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContainerWidget2</class>
 <widget class="QWidget" name="ContainerWidget2">
  <property name="windowTitle">
   <string>ContainerWidget2</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>181</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>青鸟</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>241</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:10px; margin-bottom:10px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:29px; background-color:#ffffff;&quot;&gt;&lt;a name=&quot;best-content-1070300192&quot;&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'PingFang SC,Lantinghei SC,Microsoft YaHei,arial,宋体,sans-serif,tahoma'; font-size:16px; color:#333333; background-color:#ffffff;&quot;&gt;羽&lt;/span&gt;&lt;span style=&quot; font-family:'PingFang SC,Lantinghei SC,Microsoft YaHei,arial,宋体,sans-serif,tahoma'; font-size:16px; color:#333333; background-color:#ffffff;&quot;&gt;ばたいたら戻らないと言って &lt;/span&gt;&lt;span style=&quot; font-family:'PingFang SC,Lantinghei SC,Microsoft YaHei,arial,宋体,sans-serif,tahoma'; font-size:16px; color:#333333;&quot;&gt;&lt;br /&gt;目指したのは苍い苍いあの空 &lt;br /&gt;悲しみはまだ覚えられず &lt;br /&gt;切なさは今つかみ始めた &lt;br /&gt;あなたへと抱くこの感情も &lt;br /&gt;今言叶に代わってく &lt;br /&gt;未知なる世界の梦から目覚めて &lt;br /&gt;この羽を広げ飞び立つ &lt;br /&gt;羽ばたいたら戻らないと言って &lt;br /&gt;目指したのは白い白いあの云 &lt;/span&gt;&lt;img src=&quot;https://gss0.bdstatic.com/70cFsjip0QIZ8tyhnq/img/iknow/qb/select-search.png&quot; /&gt;&lt;a name=&quot;selectsearch-icon&quot;&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'PingFang SC,Lantinghei SC,Microsoft YaHei,arial,宋体,sans-serif,tahoma'; font-size:16px; color:#333333;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'PingFang SC,Lantinghei SC,Microsoft YaHei,arial,宋体,sans-serif,tahoma'; font-size:16px; color:#333333;&quot;&gt;突き抜けたら见つかると知って &lt;br /&gt;振り切るほど &lt;br /&gt;苍い苍いあの空 苍い苍いあの空 苍い苍いあの空 &lt;br /&gt;&lt;br /&gt;爱想尽きたよう音で &lt;br /&gt;锖びれた古い窓は壊れた &lt;br /&gt;见饱きたかごは &lt;br /&gt;ほら舍てていく 振り返ることはもうない &lt;br /&gt;高鸣る鼓动に 呼吸を预けて &lt;br /&gt;この窓をけって 飞び立って &lt;br /&gt;&lt;br /&gt;駆け出したら 手にできるといって &lt;br /&gt;いざなうのは 远い远いあの声 &lt;br /&gt;まぶしすぎた あなたの手も握って &lt;br /&gt;求めるほど 青い青いあの空 &lt;br /&gt;&lt;br /&gt;落ちていくと 分かっていた &lt;br /&gt;それでも 光を追い続けていくよ &lt;br /&gt;&lt;br /&gt;羽ばたいたら 戻らないといって &lt;br /&gt;探したのは 白い白いあの云 &lt;br /&gt;つきぬけたら 见つかると知って &lt;br /&gt;振り切るほど 青い青いあの空 &lt;br /&gt;青い青いあの空 &lt;br /&gt;青い青いあのそら&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>这里是用来看省略模式的</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>101</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>199</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Page 1</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_flat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tool Button</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>199</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Page 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>QtabWidget</string>
   </property>
   <widget class="QComboBox" name="combo_elide">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_tabposition">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_tabshape">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_document">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>documentMode</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_moveable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>moveable</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_closable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>tabsClosable</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_autohide">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>tabBarAutoHide</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_scrollBtns">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>userScrollButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>elideModel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>tabPosition</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>tabShape</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QPushButton" name="btn_addItem">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>插入一个新的item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delItem">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除当前item</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
